modules/users/server/routes/admin.server.routes.js

```diff
@@ -7,15 +7,18 @@ var adminPolicy = require('../policies/admin.server.policies'),
 	admin = require('../controllers/admin.server.controller');
 
 module.exports = function (app) {
+	// User route registration first
+	require('./users.server.routes.js')(app);
```

Comment:
This seems like the glue, that holds this solution together. If that's the case, I would comment that a bit more; possibly explaining why this is being done.